scripts/zowe-parse-yaml.sh

```diff
@@ -196,93 +201,98 @@ parseConfiguationFile ./zowe-install.yaml
 if [[ $ZOWE_ROOT_DIR == "" ]] 
 then
     ZOWE_ROOT_DIR="~/zowe/$ZOWE_VERSION"
-    echo "  ZOWE_ROOT_DIR not specified:  Defaulting to ~/zowe/$ZOWE_VERSION"
+    echo "  ZOWE_ROOT_DIR not specified:  Defaulting to ~/zowe/$ZOWE_VERSION" >> $LOG_FILE
```

Comment:
These errors should be written to the log, but should also come out on stdout so the user can see that their configuration is incorrect

Reply:
OK, I will do that.